core/charm/seriesselector.go.use

```diff
@@ -28,7 +28,7 @@ type modelConfig interface {
 }
```

Comment:
What's going on with this file name? Guessing it should be deleted?

Reply:
deleting, I forgot and did the wrong `git add`